Tribler/Core/Utilities/utilities.py

```diff
@@ -171,7 +171,7 @@ def isValidTorrentFile(metainfo):
         validTorrentFile(metainfo)
         return True
     except ValueError:
-        print_exc()
+        logger.exception("Could not check torrentfile: a ValueError was thrown", exc_info=True)
```

Comment:
Ah you are right. "
Logger.exception(msg, *args, **kwargs)

    Logs a message with level ERROR on this logger. The arguments are interpreted as for debug(), except that any passed exc_info is not inspected. Exception info is always added to the logging message. This method should only be called from an exception handler." 

Removing it now :p